packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -200,7 +213,11 @@ export class GCFBootstrapper {
         taskId
       );
 
-      logger.metric(buildTriggerInfo(triggerType, id, request.body));
+      const triggerInfo = buildTriggerInfo(triggerType, id, request.body);
+      logger.metric(triggerInfo);
+
+      delete triggerInfo.message; // we don't want to bind the message to every log entry
+      GCFBootstrapper.bindPropertiesToLogger(triggerInfo);
```

Comment:
Could we make `buildTriggerInfo` not include `message`, set the logger to the child logger, then do `logger.metric("message originally in buildTriggerInfo")`?

Reply:
You mean like this?

```ts
      const triggerInfo = buildTriggerInfo(triggerType, id, request.body);
      GCFBootstrapper.bindPropertiesToLogger(triggerInfo); // no message in triggerInfo

      logger.metric({ 
        message: `Execution started by ${triggerType}`, 
        ...triggerInfo
      });
```
This works too!

Reply:
Since the trigger info should already be added to all the log entries, you should only have to call 
```typescript
logger.metric(`Execution started by ${triggerType}`);
```

Reply:
ahh ok ok - makes sense

Reply:
done!